docs/recipes/reducers/04-UsingCombineReducers.md

```diff
@@ -0,0 +1,111 @@
+# Using `combineReducers`
+
+## Core Concepts
+
+
+The most common state shape for a Redux app is a plain Javascript object containing "slices" of domain-specific data at each top-level key.  Similarly, the most common approach to writing reducer logic for that state shape is to have "slice reducer" functions, each with the same `(state, action)` signature, and each responsible for managing all updates to that specific slice of state.  Multiple slice reducers can respond to the same action, independently update their own slice as needed, and the updated slices are combined into the new state object.
+
+Because this pattern is so common, Redux provides the `combineReducers` utility to implement that behavior.  It is an example of a _higher-order reducer_, which takes an object full of slice reducer functions, and returns a new reducer function.
+
+There are several important ideas to be aware of when using `combineReducers`:
+
+- First and foremost, `combineReducers` is simply **a utility function to simplify the most common use case when writing Redux reducers**.  You are *not* required to use it in your own application, and it does *not* handle every possible scenario.  It is entirely possible to write reducer logic without using it, and it is quite common to need to write custom reducer logic for cases that `combineReducer` does not handle.  (See [Beyond `combineReducers`](./05-BeyondCombineReducers.md) for examples and suggestions.)  
```

Comment:
Put an anchor here so that we can direct link to it every time someone opens a new issue to add new behavior to `combineReducers`. 😉